packages/vue/src/components/molecules/SfCard/SfCard.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="sf-card">
+    <SfImage :src="image" class="sf-card__image" />
+    <!-- @slot Custom content for heading and description-->
+    <slot name="details" v-bind="{ title, description, titleLevel }">
+      <div class="sf-card__details">
+        <SfHeading
+          :title-level="titleLevel"
+          :title="title"
+          class="sf-card__title"
+        />
+        <p class="sf-card__description">
+          {{ description }}
+        </p>
+      </div>
+    </slot>
+    <!-- @slot Slot content will replace button -->
+    <slot name="action">
+      <SfButton :link="link" class="sf-card__action" v-on="$listeners">
+        {{ buttonText }}
+      </SfButton>
+    </slot>
+  </div>
+</template>
+<script>
+import SfButton from "../../atoms/SfButton/SfButton.vue";
+import SfHeading from "../../atoms/SfHeading/SfHeading.vue";
+import SfImage from "../../atoms/SfImage/SfImage.vue";
+
+export default {
+  name: "SfCard",
+  components: {
+    SfButton,
+    SfHeading,
+    SfImage,
+  },
+  props: {
+    /**
+     * Card image
+     */
+    image: {
+      type: [String, Object],
```

Comment:
since `sfimage` `src` doesn't accept `Object` anymore, please remove `Object` from here. We will have to improve this later on to match the new syntax but urgent